charmcraft/parts/reactive.py

```diff
@@ -107,7 +108,7 @@ class ReactivePlugin(plugins.Plugin):
     properties_class = ReactivePluginProperties
     validator_class = ReactivePluginEnvironmentValidator
 
-    @override
+    @overrides.override
```

Comment:
why did you use typing override and then switch to this?

Reply:
Working towards consistency with snapcraft/rockcraft, and this provides more/better functionality than the implementation in `typing`